README.md

```diff
@@ -9,3 +9,32 @@ For more information about the control container, including how to use it and ho
 
 You'll need Docker 17.06.2 or later, for multi-stage build support.
 Then run `make`!
+
+## Connecting to AWS Systems Manager (SSM)
+
+Starting from v0.5.0, users have the option to pass in their own activation information for SSM.
+This is for users that want to set up on-premises virtual machines (VMs) in their hybrid environment as managed instances.
+
+Users can add their [own activations](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-managed-instance-activation.html) by populating the control container's user data with a base64-encoded JSON block.
+
+To use hybrid activations for managed instances you will want to generate a JSON-structure like this:
+
+```
+{
+  "ssm": {
+    "activation-id": "foo",
+    "activation-code": "bar",
+    "region":"us-west-2"
```

Comment:
These names aren't consistent with the user data used in the admin container (`foo-bar` vs `foo_bar`). Was this intentional? As much as we can, I'd prefer that the "convention" stays consistent for users. This aligns with Bottlerocket's `user-data.toml` naming (which uses skewers) so one *could* argue that this one is the "right way" and the admin is in the wrong.. but we already have the admin change out, so I'm more inclined to align with that user data as this user data and the admin's, to me, roughly feels like it'll be treated as same namespace and with similar conventions by end users.

Reply:
Based on the other settings I see when I run `apiclient -u /settings` I would say that the admin container is the weird one, and that `foo-bar` is the "correct" convention, but I do not have strong feelings. I think the it might be worth getting some additional opinions. I think the admin container is the way it is because the authorized keys are typically stored in a file named `authorized_keys`, but that may have been a poor decision on my part. To make this conundrum even more fun, when you generate an activation token with `aws ssm create-activation` it returns JSON with a `FooBar` convention.

Reply:
After some offline discussion, we're going to keep the admin container convention and use `foo_bar` (snake_case) for the settings inside host-container user data in general. I'll add this change.

Reply:
I'm in favor of sticking with the underscore (`foo_bar`) implementation both here and the admin container. We can revisit this at a later date if we want to _also_ provide support for skewers (`foo-bar`) to more closely match our Bottlerocket user data.